Formula/r.rb

```diff
@@ -39,16 +49,34 @@ def install
     args = [
       "--prefix=#{prefix}",
       "--enable-memory-profiling",
-      "--without-cairo",
-      "--without-x",
       "--with-tcl-config=#{Formula["tcl-tk"].opt_lib}/tclConfig.sh",
       "--with-tk-config=#{Formula["tcl-tk"].opt_lib}/tkConfig.sh",
-      "--with-aqua",
       "--with-blas=-L#{Formula["openblas"].opt_lib} -lopenblas",
       "--enable-R-shlib",
       "--disable-java",
     ]
 
+    on_macos do
+      args << "--without-cairo"
```

Comment:
Could probably save this for another PR, but it might be nice if we could figure out how to enable cairo support on macOS. I remember @jonchang mentioning this would add functionality, plus minimising the macOS-Linux diff would help maintainability.

Reply:
Let me see if it's an easy fix...

Reply:
OK, that was easier than I thought. I've also included a simple test to confirm that the R runtime recognizes `cairo`, though testing whether it'll actually work is beyond my abilities. :grin: